main.py

```diff
@@ -3,14 +3,31 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
+        clock.tick(60) 
+        # Event handling to check if the user wants to quit
```

Comment:
**Suggestion:** Enhance event handling for player interactions.

**Existing Code:**
```Python
for event in pygame.event.get():
    if event.type == pygame.QUIT:
        pygame.quit()
        return
```

**Improved Code:**
```Python
for event in pygame.event.get():
    if event.type == pygame.QUIT:
        pygame.quit()
        return
    elif event.type == pygame.KEYDOWN:
        # Handle key presses for player movement or actions
```

**Details:**
Consider adding a mechanism to handle other events (like keyboard input) in the game loop for player interaction.